sql/core/src/test/scala/org/apache/spark/sql/internal/SQLConfSuite.scala

```diff
@@ -319,6 +320,11 @@ class SQLConfSuite extends QueryTest with SharedSparkSession {
     assert(e2.message.contains("Cannot modify the value of a static config"))
   }
 
+  test("SPARK-36643: Cannot modify the value of a Spark config") {
+    val e1 = intercept[AnalysisException](spark.conf.set("spark.driver.host", "myhost"))
+    assert(e1.message.contains("Cannot modify the value of a Spark config"))
```

Comment:
can you check if "http" exists?

Reply:
Yes, http and https both exists.

Reply:
no I mean can we test if `http` exists in the error message?

Reply:
You mean to rewrite the test as below, right?

```
test("SPARK-36643: Show migration guide when attempting SparkConf") {
    val e1 = intercept[AnalysisException](spark.conf.set("spark.driver.host", "myhost"))
    assert(e1.message.contains("http"))
  }
```